tests/homePage/HotSellersSection.spec.js

```diff
@@ -0,0 +1,35 @@
+// @ts-check
+import { test, expect } from "@playwright/test";
+
+test("Six items displayed at hot sellers", async ({ page }) => {
+  await page.goto("https://magento.softwaretestingboard.com/");
+  const productItems = await page.$$(".product-item");
+  expect(productItems).toHaveLength(6);
+});
+
+test("Item redirects correctly on home page", async ({ page }) => {
+  await page.goto("https://magento.softwaretestingboard.com/");
+
+  const productItems = await page.$$(".product-item");
+  const priceItems = await page.locator(".price").allTextContents();
+  const nameItems = await page.locator(".product-item-link").allInnerTexts();
+
+  for (let item in productItems) {
+    // Re-fetch the elements inside the loop to ensure they are still attached to the DOM
+    const updatedProductItems = await page.$$(".product-item");
+
+    // Go to item`s page
+    await updatedProductItems[item].click();
+
+    // get the name of the item
```

Comment:
Capitalize the first letter from the comment in all the comments. Make it more consistent.